bin/compile

```diff
@@ -76,6 +77,7 @@ else
   mkdir -p "$APT_SOURCELIST_DIR"   # make dir for sources
```

Comment:
I think we need to add a `mkdir -p "${APT_SOURCEPARTS_DIR}"` after this line?

Also, I'm curious how the tests are passing without it?

Reply:
When the source parts directory files are copied it's with:
```
cp -R "/etc/apt/sources.list.d" "$APT_SOURCEPARTS_DIR"
```
And that will create the `sources.list.d` because it's recursive and parent directory `$APT_SOURCELIST_DIR` has already been created.  At least that's how it seems to work when I do a similar operation locally.

Reply:
Ah yes! Sorry I think I read it as `/etc/apt/sources.list.d/*` or similar :-)